docker/build.sh

```diff
@@ -13,6 +13,19 @@
 set -ex
 PROJECT=rawls
 
+function build_check()
+{
+    HASH_TAG=${GIT_SHA:0:12}
+    docker_pull_return=0
+    docker pull $REPO:${HASH_TAG} || docker_pull_return=$?
```

Comment:
Do we need to do a pull here?  That could take time too.  Is there a command to simply check that the tag exists?

Reply:
good thought. i am thinking of using the dockerhub api to check, which means we also don't have to run a docker command and thus can run this script *in* a docker and thus move it to a common location